toolkit/sidechain/pallet/src/lib.rs

```diff
@@ -1,6 +1,125 @@
+//! Pallet that establishes a Partner Chain as a Cardano sidechain
+//!
+//! # Purpose of this pallet
+//!
+//! This pallet serves as the starting point for building a Partner Chain runtime.
+//! It stores its genesis UTXO which serves as its global identifier and divides
+//! Partner Chain slots into epoch.
```

Comment:
Cosmetic: "epochs".
I would even go with "divides slots into Partner Chain epochs". Slots are from polkadot-sdk in fact, but epoch notion comes from us